packages/nocodb/src/controllers/form-columns.controller.ts

```diff
@@ -19,10 +26,13 @@ export class FormColumnsController {
   async columnUpdate(
     @Param('formViewColumnId') formViewColumnId: string,
     @Body() formViewColumnbody: FormColumnUpdateReqType,
+
+    @Request() req: any,
```

Comment:
can use `RequestType` instead of `any`? (apply to all)

Reply:
I will update this part as well in a followup pr... and inside hook I will use a custom type since in some case only certain fields we are passing ( user, headers, etc... )